pkg/scanners/scannerv4/scannerv4.go

```diff
@@ -202,3 +208,92 @@ func (s *scannerv4) GetVulnerabilities(image *storage.Image, components *types.S
 
 	return imageScan(image.GetMetadata(), vr), nil
 }
+
+func (s *scannerv4) GetNodeVulnerabilityReport(node *storage.Node, indexReport *v4.IndexReport) (*v4.VulnerabilityReport, error) {
+	nodedigest, err := name.NewDigest("registry/repository@sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
```

Comment:
I remember the story with custom sha, but the readers of the code may not remember it. Let's put it into a constant and explain in a comment why we use a hardcoded value.

2nd thought: is `aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa` a valid sha256 value?

Reply:
This is resolved but looks like it is not addressed. Let me unresolve it for now.